.github/workflows/packaging.yml

```diff
@@ -51,3 +51,22 @@ jobs:
 
       - name: Create packages
         run: ./packpack/packpack
+
+      - name: Deploy packages
+        # We need this step to run only on push with tag 
```

Comment:
Extra space at the end.
Set a dot at the end of the sentence.
https://github.com/tarantool/tarantool/wiki/Code-review-procedure#code-style
`Start sentences from a capital letter, end with a dot...`

Reply:
Fixed.